packages/test/test-end-to-end-tests/src/test/attachLifecycle.spec.ts

```diff
@@ -31,9 +31,9 @@ const sharedPoints = [3, 4, 5];
 const ddsKey = "string";
 
 const testConfigs = generatePairwiseOptions({
```

Comment:
Curious if you put a breakpoint on the changed line to see the specific generated pairwise cases that hit it?